homeassistant/components/abode/__init__.py

```diff
@@ -29,7 +29,7 @@
 
 CONF_POLLING = "polling"
 
-DEFAULT_CACHEDB = "./abodepy_cache.pickle"
+DEFAULT_CACHEDB = ".abodepy.pickle"
```

Comment:
This is a breaking change as now the old file cannot be found. What is the impact of this ?

Reply:
Ah, this is a very good point. When you set up an integration in the UI does it store a username and password somewhere?

The pickle file, strictly speaking, isn't required other than to store a UUID to pass to the abode API. Since abodepy does not handle 2FA the UUID isn't actually needed for anything other than to make the API happy. This wouldn't have been a breaking change with the yaml configuration because it would just use the username and password from the configuration and regenerate the pickle file with a new UUID (which, as previously said, wasn't doing anything important).

Perhaps it would be best to just keep the old filename `abodepy_cache.pickle` and get rid of the `./` part...

Reply:
yes. that will not break anything.

Reply:
I'll change it back then :-)